### ISSUE TITLE
Photo repeat displayed in ArcGIS Online 

### ISSUE THREAD

**Issue by esauthier** [Esri/Survey123Community#269]
Hi there, 
I am wondering if it's possible to see the pictures taken with the option "photo repeat" in ArcGIS Online. If yes, do we have to activate a particular option? If yes, can we download them back from the cloud for a backup for example, how? 

Thank you a lot!
Eulalie

**Comment by survey123** [comment]
Repeats are modeled as a table or layer that is related to the main feature layer of your Survey. Unfortunately, ArcGIS Online does not support displaying attachments in related tables.

BUG-000086534 : Photo attachments in a related table are visible in Collector for ArcGIS but invisible when viewing the related records in the ArcGIS Online map viewer.

**Comment by esauthier** [comment]
Can we download the photo coming from a photo repeat  from the cloud for a backup for example, as we can do for a simple photo in AGOL?If yes, what is the procedure?

**Comment by survey123** [comment]
Adding to the September 2016 update.
Plan is to support the multiline appearance in image questions. When used, you will be able to capture as many photos as you like without having to use a repeat. In this way, your attachments will be much easier to work with from Web Maps and ArcGIS for Desktop.

**Comment by simontheguide** [comment]
Has the September release with the multiline appearance for image questions been released yet? Cheers.

**Comment by IsmaelInRedlands** [comment]
Nope. It will not make it to September. Currently planned for October. We have been going back and forth on this one. Still working on this.

**Comment by simontheguide** [comment]
Awesome, thanks.

I thought that I was able to view attachments in a related table in an earlier version of the survey123 in ArcGIS online?


Simon Allard | GIS Analyst | Bay of Plenty Regional Council | Tauranga, New Zealand | Ph: 0800 884 881 x8219 | Web: www.boprc.govt.nz<http://www.boprc.govt.nz>
Please consider the environment before printing this email
